apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.ts

```diff
@@ -472,6 +472,17 @@ export class AdvanceReceiptAdjustmentComponent implements OnInit {
      */
     public calculateTax(entryData: Adjustment, index: number): void {
         let entry: Adjustment = cloneDeep(entryData);
+        let selectedItem;
+        if (entryData && this.newAdjustVoucherOptions && this.newAdjustVoucherOptions.length) {
+            selectedItem = this.newAdjustVoucherOptions.find(item => {
+                return item.label === entryData.voucherNumber;
+            });
+            // To restrict user to enter amount less or equal selected voucher amount
+            if (selectedItem && selectedItem.additional && selectedItem.additional.dueAmount && this.adjustVoucherForm.adjustments[index].dueAmount.amountForAccount > selectedItem.additional.dueAmount.amountForAccount) {
```

Comment:
Nullity check is not added for `this.adjustVoucherForm.adjustments[index].dueAmount.amountForAccount`

Reply:
No need to check nullity actually we are calling this method 
this.adjustVoucherForm.adjustments[index].dueAmount input box blur funtion
which is from this.adjustVoucherForm.adjustments